tests/script-based-pre/stubbing_compiler_sessions/expected

```diff
@@ -0,0 +1 @@
+VERIFICATION:- SUCCESSFUL
```

Comment:
Is this `expected` file used? Isn't `check_compiler_sessions.expected` sufficient?

Reply:
yes... this is garbage. Let me remove